dashboards/localvolumeprovisioner.yaml

```diff
@@ -0,0 +1,470 @@
+                "alignLevel": null
+             }
+          },
+          {
+             "cacheTimeout": null,
+             "gridPos": {
+                "h": 8,
+                "w": 12,
+                "x": 0,
+                "y": 9
+             },
+             "id": 8,
+             "interval": "15s",
+             "links": [ ],
+             "options": {
+                "maxValue": 100,
+                "minValue": 0,
+                "orientation": "auto",
+                "showThresholdLabels": false,
+                "showThresholdMarkers": true,
+                "thresholds": [
+                   {
+                      "color": "green",
+                      "index": 0,
+                      "value": null
+                   },
+                   {
+                      "color": "red",
+                      "index": 1,
+                      "value": 80
+                   }
+                ],
+                "valueMappings": [ ],
+                "valueOptions": {
+                   "decimals": null,
+                   "prefix": "",
+                   "stat": "last",
+                   "suffix": "",
+                   "unit": "none"
+                }
+             },
+             "pluginVersion": "6.1.6",
+             "targets": [
+                {
+                   "expr": "local_volume_provisioner_proctable_running{instance=~\"$instance\"}",
+                   "format": "time_series",
+                   "interval": "",
+                   "intervalFactor": 2,
+                   "legendFormat": "running",
+                   "refId": "A"
+                }
+             ],
+             "timeFrom": null,
+             "timeShift": null,
+             "title": "Running proctables",
+             "type": "gauge"
+          },
+          {
+             "aliasColors": { },
+             "bars": false,
+             "dashLength": 10,
+             "dashes": false,
+             "fill": 1,
+             "gridPos": {
+                "h": 8,
+                "w": 12,
+                "x": 0,
+                "y": 17
+             },
+             "id": 4,
+             "interval": "15s",
+             "legend": {
+                "avg": false,
+                "current": false,
+                "max": false,
+                "min": false,
+                "show": true,
+                "total": false,
+                "values": false
+             },
+             "lines": true,
+             "linewidth": 1,
+             "links": [ ],
+             "nullPointMode": "null as zero",
+             "percentage": false,
+             "pointradius": 2,
+             "points": false,
+             "renderer": "flot",
+             "seriesOverrides": [ ],
+             "spaceLength": 10,
+             "stack": false,
+             "steppedLine": true,
+                "targets": [
+                {
+                   "expr": "idelta(local_volume_provisioner_apiserver_requests_total{instance=~\"$instance\"}[1m])",
```

Comment:
The metrics are all cumulative, so some kind of difference function is needed to present the state of the provisioner of a given time. We have several other choices here:
`rate`: The end results would be a floating point number that's very hard to understand.
`increase`: This is better than `rate` since it gives the difference within the time interval. However this function will do extrapolation, which would end up with weird floating point numbers that does't match the heatmap.
`delta`: This function also gives similar results to `increase`. But if there's only one single value in a given duration the data will be hidden.

Therefore I ended up with `idelta`, where each point on the graph consumes two data points, and that's why I set the resolution to 1/2. This makes the request graph match the heatmaps and gives the exact number of requests. It also works if there are consecutive nonzero data points.

That said, if we don't need to show the histogram heatmaps, then we don't need this.